src/chrono.rs

```diff
@@ -0,0 +1,65 @@
+//! chrono types
+//!
+//! based on Dart VM, microseconds unit is used
+
+use crate::{ffi::DartCObject, IntoDart};
+
+impl IntoDart for chrono::DateTime<chrono::Utc> {
+    /// on the other side of FFI, value should be reconstructed like:
+    ///
+    /// - hydrate into Dart [DateTime](https://api.flutter.dev/flutter/dart-core/DateTime/DateTime.fromMicrosecondsSinceEpoch.html)
```

Comment:
When you mention something in the docs, please refer it to a fixed version of the Dart std, and better be dart std not from flutter website.

```suggestion
    /// - hydrate into Dart [DateTime](https://api.dart.dev/stable/2.18.0/dart-core/DateTime/DateTime.fromMicrosecondsSinceEpoch.html)
```

Reply:
ok in c3cd8dd35944fba13586382d7be26679cd533980